src/docs/product/performance/retention-priorities/index.mdx

```diff
@@ -83,6 +83,42 @@ Some projects within your organization are very high volume compared to others.
 Transactions can be logically grouped by their name within a project. This means that transactions with the same name most likely indicate the same operation or code path, while transactions with different names usually indicate different operations.
 We prioritize low-volume transactions for the same reason we prioritize low-volume projects – so they don't get drowned out by high-volume transactions. We want to make sure that transactions representing low-volume code paths are sampled in enough quantities to generate a complete view of your application.
 
+### Investigation Mode
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+Our automated retention priorities work well generically in order to collect a baseline of samples, because they prioritize retaining data that is valuable for any customer at any given time. But there are certain scenarios, such as investigating a particular issue, when very specific data becomes more important temporarily. In such cases, sometimes the automated sampling priorities do not ensure sufficient or even any samples to use during this investigation.
```

Comment:
```suggestion
Our automated retention priorities work well in a generic manner to collect a baseline of samples because they prioritize retaining data that is valuable for any customer at any given time. However, there are certain scenarios, such as investigating a particular issue, when very specific data becomes temporarily more important. In such cases, sometimes the automated sampling priorities do not provide enough samples, so you may want to create investigation rules.
```